grpc/test_proto/interceptors_test.proto

```diff
@@ -2,13 +2,15 @@ syntax = "proto3";
 
 option go_package = "github.com/letsencrypt/boulder/grpc/test_proto";
 
+import "google/protobuf/duration.proto";
+
 service Chiller {
   // Sleep for the given amount of time, and return the amount of time slept.
   rpc Chill(Time) returns (Time) {}
 }
 
 message Time {
-  // Next unused field number: 2
-   // TODO (#7097) Change this to google.protobuf.Duration
+  // Next unused field number: 3
   int64 timeNS = 1; // Unix timestamp (nanoseconds)
+  google.protobuf.Duration time = 2;
```

Comment:
This code is only used in tests, so we don't need to do the "two fields" shuffle. We can delete the old `timeNS`.

Also, really the message and the field should be named `Duration` and `duration` respectively.

Reply:
Good idea, and yeah that will help simplify one more step.